libs/designsystem/src/lib/components/modal/services/modal.model.ts

```diff
@@ -1,6 +1,11 @@
 import { KirbyAnimation } from '../../../animation/kirby-animation';
 
-export interface Modal {
+export interface IModal {
```

Comment:
Consider not prefixing interfaces with I, since it's not recommended by the TypeScript team.

Reply:
Yep, agree, but if I don't then the `Modal` and `IModal` have to share props + methods.

Reply:
What about calling it `ModalWindow` or `ModalWrapper`?